<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainQCCheckPage</class>
 <widget class="QWidget" name="MainQCCheckPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>430</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainQCCheckPage {	
	border-image: url(&quot;:rc/res/main_bk.png&quot;) 50 00 0 0 repeat;
}

QPushButton {
	border: 1px solid  rgb(0, 0, 0);
	border-radius: 0px;
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0.505682 rgba(35, 97, 222, 247), stop:0.539773 rgba(61, 114, 223, 247), stop:1 rgba(93, 142, 239, 238));
	color: rgb(255, 255, 255);
	font: 16px;
}

#tableWidgetOrderInfo {
border-radius: 0px;	
background-color: rgba(123, 187, 255, 100);
color: rgb(255, 255, 255);
font: bold 16pt ;
border-radius: 9px;
}

#tableWidgetQCList {
border: 0px solid  rgb(85, 85, 85);
color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 0);
margin-top: 4px;
}

#tableWidgetQCList::item { 
	color: rgb(255, 255, 255);
	font: bold 18px;
	margin: 2px ;
	border: 0px solid  rgb(255, 255, 255);
	border-right: 5px solid  rgb(29, 29, 29);
	border-bottom: 2px solid  rgb(29, 29, 29);
	background-color: rgba(88, 188, 255, 100);
}

#tableWidgetQCList::item:selected {
    margin: 0px ;
	background-color: rgba(88, 188, 255, 200);
}
#tableWidgetQCList::item:hover {
    margin: 0px ;
	background-color: rgba(88, 188, 255, 200);
}
#labelQCTitle {
	font: bold 16px;
	color: rgb(255, 255, 255);
}
#btPageLast{border:0px solid rgb(220, 220, 220);
border-bottom:3px solid rgb(16, 16, 16);
border-top-left-radius:20px;
}
#btPageNext{border:0px solid rgb(220, 220, 220);
border-bottom:3px solid rgb(16, 16, 16);
border-top-right-radius:20px;}</string>
  </property>
  <widget class="QPushButton" name="btPageLast">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>上一页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btPageNext">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>下一页</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetQCList">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>501</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectItems</enum>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideRight</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::NoPen</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>6</number>
   </property>
   <property name="columnCount">
    <number>4</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>124</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>62</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>21</number>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>值</string>
    </property>
   </column>
   <column/>
   <column/>
   <item row="0" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelQCTitle">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>第?件 次品原因 30种 共3页 第1页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btSave">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>380</y>
     <width>91</width>
     <height>47</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>47</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-right-radius:10px;
border-bottom-right-radius:10px;</string>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btUpItem">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>91</width>
     <height>47</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>47</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-left-radius:10px;
border-bottom-left-radius:10px;</string>
   </property>
   <property name="text">
    <string>上一件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btNextItem">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>91</width>
     <height>47</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>47</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>下一件</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetOrderInfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>361</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectItems</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::NoPen</enum>
   </property>
   <property name="rowCount">
    <number>9</number>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>109</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>38</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>属性</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>数值</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>派工单号</string>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignLeft|AlignVCenter</set>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>总件/当前</string>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string>工单号</string>
    </property>
    <property name="textAlignment">
     <set>AlignLeft|AlignVCenter</set>
    </property>
   </item>
   <item row="3" column="0">
    <property name="text">
     <string>产品编号</string>
    </property>
    <property name="textAlignment">
     <set>AlignLeft|AlignVCenter</set>
    </property>
   </item>
   <item row="4" column="0">
    <property name="text">
     <string>产品描述</string>
    </property>
    <property name="textAlignment">
     <set>AlignLeft|AlignVCenter</set>
    </property>
   </item>
   <item row="5" column="0">
    <property name="text">
     <string>派工数量</string>
    </property>
    <property name="textAlignment">
     <set>AlignLeft|AlignVCenter</set>
    </property>
   </item>
   <item row="6" column="0">
    <property name="text">
     <string>本班生产总数</string>
    </property>
    <property name="textAlignment">
     <set>AlignLeft|AlignVCenter</set>
    </property>
   </item>
   <item row="7" column="0">
    <property name="text">
     <string>本班次品总数</string>
    </property>
    <property name="textAlignment">
     <set>AlignLeft|AlignVCenter</set>
    </property>
   </item>
   <item row="8" column="0">
    <property name="text">
     <string>本班次品率</string>
    </property>
    <property name="textAlignment">
     <set>AlignLeft|AlignVCenter</set>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
